management/global/organizations/aws-tf-organization.md

```diff
@@ -0,0 +1,99 @@
+## AWS Terraform Management
+
+### Update AWS Organization features
+1. Go to `management/global/organizations`.
+2. Edit `organization.tf` to enable/disable Organization features
+3. Or edit the `policies_scp.tf` file to add/remove/update Service Control Policies.
+4. Finally, run the Terraform workflow to apply the actual changes.
```

Comment:
This is a subtle observation. May be the phrase "Finally, run the Terraform workflow to apply the actual changes." can be expressed as "Finally, run the Terraform workflow to actually apply the changes."?

Reply:
this is minimum... but the same usage of "actual" can be seen later in this same text

Reply:
@juanmatias suggested commit accepted, thanks!